imports/plugins/core/accounts/client/components/accountsDashboard.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import AccountsTable from "./accountsTable";
+import AccountsManageContainer from "../containers/accountsManageContainer";
+
+class AccountsDashboard extends Component {
+  static propTypes = {
+    accounts: PropTypes.array,
+    groups: PropTypes.array
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      accounts: props.accounts,
+      groups: props.groups,
+      showSideBar: false,
+      selectedGroup: {}
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const { groups, accounts } = nextProps;
+    const selectedGroup = groups.find(grp => grp._id === (this.state.selectedGroup || {})._id);
+    this.setState({ groups, accounts, selectedGroup });
+  }
+
+  handleGroupSelect = group => {
+    this.setState({ showSideBar: true, selectedGroup: group });
+  };
+
+  tableClassName() {
+    if (this.state.showSideBar) {
+      return "col-md-9";
+    }
+    return "col-md-12";
+  }
+
+  detailDivClassName() {
+    if (this.state.showSideBar) {
+      return "col-md-3";
+    }
+    return "hide";
+  }
+
+  renderGroupDetail = () => {
+    if (this.state.showSideBar) {
+      const { groups, accounts } = this.state;
+      return (
+        <AccountsManageContainer
+          className="accounts-manage-container"
+          group={this.state.selectedGroup}
+          groups={groups}
+          accounts={accounts}
+          onChangeGroup={this.handleGroupSelect}
+        />
+      );
+    }
+    return null;
+  };
+
+  renderGroupsTable(groups) {
+    if (Array.isArray(groups)) {
+      return groups.map((group, index) => {
+        return (
+          <div key={index}>
```

Comment:
Is this `<div>` necessary? you can add `key` to the `<AccountsTable />`. If it necessary, put a `className` on it like `rui accounts-table-container` or something appropriate.